### PR TITLE
Config tweaks

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: concepts/src/concept_csv_export.py [hunk 1]
```diff
@@ -22,10 +22,11 @@
 # Globals -- modified only during initialization
 VERBOSE = False
 DOCKER = False
+DOCKER_CONTAINER = "openmrs-sdk-mysql"
 VERSION = "2.3"
 LOCALES = ["en"]
 DEFAULT_LOCALE = "en"
-ENCODING = "UTF-8"
+ENCODING = "unicode_escape"
```
Comment (mogoodrich) [inline concepts/src/concept_csv_export.py:29]: Not sure exactly what this does, but UTF-8 wasn't working for me, and I found this via Google.

Comment (mseaton) [reply concepts/src/concept_csv_export.py:29]: Interesting @mogoodrich .  You'll notice that @brandones originally had 'latin-1' for this, and changed it to 'UTF-8' on review here:  https://github.com/PIH/iniz-exporters/pull/3.  Not sure why it is failing now, but I never tried running it before...

**Review by mseaton — APPROVED** [review]

File: concepts/src/concept_csv_export.py (hunk 1 above)
Comment (mogoodrich) [reply concepts/src/concept_csv_export.py:29]: Brandon has a Mac, correct?  My wild guess is that it was some OS difference...

Comment (mogoodrich) [reply concepts/src/concept_csv_export.py:29]: (the underlying script just runs a command line and attempts to parse the results)

**Comment by mogoodrich** [comment]
Brandon, I assume you aren't planning on looking at this at this point, so merging, but post-commit comments always welcome! :)

File: concepts/src/concept_csv_export.py (hunk 1 above)
Comment (brandones) [reply concepts/src/concept_csv_export.py:29]: I run Ubuntu and was using the Dockerized MySQL that the OpenMRS SDK sets up.

Comment (mogoodrich) [reply concepts/src/concept_csv_export.py:29]: Strange, I was doing the same, the only difference was I set up the dockerized container myself....

**Pull request merged**